src/main/java/duke/command/UnknownCommand.java

```diff
@@ -2,15 +2,15 @@
 
 import duke.exception.DukeException;
 import duke.storage.Storage;
-import duke.ui.Ui;
 import duke.task.TaskList;
+import duke.ui.Ui;
 
 /**
  * Represents an UnknownCommand where user input is unclear.
  */
 public class UnknownCommand extends Command {
```

Comment:
I think it might be better to use a stronger word as as "InvalidCommand", rather than "unknown", which can give the feeling of the command being unhandled?